tests/Microsoft.Windows.Setting.Accessibility/Microsoft.Windows.Setting.Accessibility.Tests.ps1

```diff
@@ -114,6 +114,35 @@ Describe 'MousePointer' {
     }
 }
 
+Describe 'DynamicScrollbars'{
+   It 'Keeps current value.'{
+      $initialState = Invoke-DscResource -Name AlwaysShowScrollbars -ModuleName Microsoft.Windows.Setting.Accessibility -Method Get -Property @{}
+
+      $parameters = @{ ShowScrollBars = 'KeepCurrentValue' }
+
+      $testResult = Invoke-DscResource -Name AlwaysShowScrollbars -ModuleName Microsoft.Windows.Setting.Accessibility -Method Test -Property $parameters
+      $testResult.InDesiredState | Should -Be $true
+
+      # Invoking set should not change these values.
+      Invoke-DscResource -Name AlwaysShowScrollbars -ModuleName Microsoft.Windows.Setting.Accessibility -Method Set -Property $parameters
+      $finalState = Invoke-DscResource -Name AlwaysShowScrollbars -ModuleName Microsoft.Windows.Setting.Accessibility -Method Get -Property @{}
+      $finalState.ShowScrollBars | Should -Be $initialState.ShowScrollBars
+   }
+
+   It 'Sets desired value.'{
+      # Randomly generate desired state. Minimum is set to 1 to avoid using KeepCurrentValue
+      $desiredScrollbarBehavior = [ShowScrollBars](Get-Random -Maximum 2 -Minimum 1)
+
+      $desiredState = @{ ShowScrollBars = $desiredScrollbarBehavior }
+
+      Invoke-DscResource -Name AlwaysShowScrollbars -ModuleName Microsoft.Windows.Setting.Accessibility -Method Set -Property $desiredState
+
+      $finalState = Invoke-DscResource -Name AlwaysShowScrollbars -ModuleName Microsoft.Windows.Setting.Accessibility -Method Get -Property @{}
+      $finalState.ShowScrollBars | Should -Be $desiredScrollbarBehavior
+	  
+   }
```

Comment:
Have it set to false initially, then try setting to true and verify that it has changed. Also check that the test method is not in desired state initally, and is in desired state after invoking set.

Reply:
Won't the differences from other resources diminish readability? Is this technical debt from removing the `enum`?